driver/src/contracts/stablex_contract.rs

```diff
@@ -193,15 +173,15 @@ fn encode_prices_for_contract(price_vector: Vec<u128>) -> (Vec<U128>, Vec<U128>)
 fn encode_execution_for_contract(
     orders: Vec<Order>,
     executed_buy_amounts: Vec<u128>,
-) -> (Vec<H160>, Vec<U128>, Vec<U128>) {
+) -> (Vec<H160>, Vec<u64>, Vec<U128>) {
     assert_eq!(
         orders.len(),
         executed_buy_amounts.len(),
         "Received inconsistent auction result data."
     );
-    let mut owners: Vec<H160> = vec![];
-    let mut order_ids: Vec<U128> = vec![];
-    let mut volumes: Vec<U128> = vec![];
+    let mut owners = vec![];
+    let mut order_ids = vec![];
+    let mut volumes = vec![];
```

Comment:
If the type declaration can be removed, what was it needed for in the first place?

Reply:
It wasn't. I removed them since I needed to change it for one of them anyway, so next time the diff is even smaller :smile: